### PR TITLE
fixed -- improved non x86_64 build times 

### PR DESCRIPTION
**Pull request by Prakashh21**
changed the base image and used conda to install those dependencies which required building from source.

in total there were  5 packages  

msgpack-python==1.0.3
MarkupSafe==2.1.1
netifaces==0.10.9
PyYAML==6.0
wrapt==1.14.0

among them I downgraded netifaces from 0.11.0 to 0.10.9 since it was not available in the conda-forge channel

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@
 # docker build --build-arg http_proxy=${PROXY} --build-arg ENVIRONMENT=local -t local/pfcon:dev .
 #
 
-FROM docker.io/library/python:3.8.12-bullseye
+FROM fnndsc/conda:latest
```
Comment (jennydaman) [inline Dockerfile:33]: Never use the `:latest` tag, please pin a version.

File: Dockerfile [hunk 2]
```diff
@@ -39,11 +39,12 @@ LABEL org.opencontainers.image.authors="FNNDSC <dev@babyMRI.org>" \
       org.opencontainers.image.source="https://github.com/FNNDSC/pfcon" \
       org.opencontainers.image.licenses="MIT"
 
+RUN conda install -y -c conda-forge msgpack-python==1.0.3 MarkupSafe==2.1.1 netifaces==0.10.9 PyYAML==6.0 wrapt==1.14.0
```
Comment (jennydaman) [inline Dockerfile:42]: conda prefers the `name=version` syntax instead of `name==version` (which is pip-specific).

Comment (jennydaman) [inline Dockerfile:42]: Please specify these dependencies in an external file, such as one created by `conda env export`, see the documentation I sent to you in chat.

**Review by jennydaman — CHANGES_REQUESTED** [review]

**Comment by Prakashh21** [comment]
I am closing this generated a new pr #125

**Pull request closed without merging**